mirrord/kube/src/api/kubernetes/seeker.rs

```diff
@@ -236,9 +237,17 @@ impl KubeResourceSeeker<'_> {
     /// Prepares [`ListParams`] that:
     /// 1. Excludes our own resources
     /// 2. Adds a limit for item count in a response
-    fn make_list_params(field_selector: Option<&str>) -> ListParams {
+    fn make_list_params(field_selector: Option<&str>, label_selector: Option<&str>) -> ListParams {
         ListParams {
-            label_selector: Some("app!=mirrord,!operator.metalbear.co/owner".to_string()),
+            label_selector: Some(join(
```

Comment:
Isn't there a `join` in std?

https://doc.rust-lang.org/std/primitive.slice.html#method.join

Reply:
Oh alright.